### PR TITLE
ISPN-10501 Various fixes for Quarkus compatibility in Server

### PR DESCRIPTION
**Pull request by wburns**
* Remove static initializer of NettyTransport to make replacment easier

https://issues.jboss.org/browse/ISPN-10501

### CODE REVIEW

File: client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/transport/netty/EPollAvailable.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import io.netty.channel.epoll.Epoll;
+
+// This is a separate class for better replacement within Quarkus as it doesn't support native EPoll
+final class EPollAvailable {
+   static private final Log log = LogFactory.getLog(EPollAvailable.class, Log.class);
+   static final boolean USE_NATIVE_EPOLL = useNativeEpoll();
+
+   private static final String USE_EPOLL_PROPERTY = "infinispan.server.channel.epoll";
+   private static final boolean IS_LINUX = System.getProperty("os.name").toLowerCase().startsWith("linux");
+   private static final boolean EPOLL_DISABLED = System.getProperty(USE_EPOLL_PROPERTY, "true").equalsIgnoreCase("false");
```
Comment (anistor) [inline client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/transport/netty/EPollAvailable.java]: This looks exactly like ```Boolean.getBoolean``` :)

**Comment by ryanemerson** [comment]
@wburns `EpollAvailabilityTest.testEpollNotAvailable` is related

File: client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/transport/netty/EPollAvailable.java (hunk 1 above)
Comment (wburns) [reply client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/transport/netty/EPollAvailable.java]: This is just what the code was before, but I agree :)

**Comment by wburns** [comment]
Test should be fixed now.

**Pull request merged**